agent/taskresource/ssmsecret/ssmsecret.go

```diff
@@ -333,7 +336,7 @@ func (secret *SSMSecretResource) retrieveSSMSecretValues(region string, names []
 	seelog.Infof("ssm secret resource: retrieving resource for secrets %v in region [%s] in task: [%s]", names, region, secret.taskARN)
 	secValueMap, err := ssm.GetSecretsFromSSM(names, ssmClient)
 	if err != nil {
-		errorEvents <- err
```

Comment:
should this be `errors.New(...)` 🤔

Reply:
errors.New only takes one string as input (func New(text string) error),  while fmt.Errorf can take extra parameters as input(func Errorf(format string, a ...interface{}) error).